docs/how_to/company/manage-cluster.md

```diff
@@ -16,28 +16,94 @@ In this step, you are required to choose the cluster's vendor and runtime servic
 In this step, you can insert some general details about your cluster:
 
 * **Cluster ID** (*required*): friendly identifier of the cluster, assigned by the user while connecting the cluster to the Console;
+* **Cluster URL** (*required*): URL used to connect to the cluster, which indicates where the cluster is exposed;
 * **Geographics**: a label used to describe the location of the cluster;
 * **Description**: a brief description of the cluster.
 
-### Step 3: Connection
+### Step 3: Credentials
 
 ![Add cluster connection 3](img/add-cluster-connection-3.png)
 
-In this step, you can insert some information about the cluster connection:
+In this step, you are required to insert some information about the cluster credentials:
 
-* **Cluster URL** (*required*): URL used to connect to the cluster, which indicates where the cluster is exposed;
-* **Proxy URL**: URL of the proxy used to connect to the cluster, usually needed when the cluster is exposed on a private network;
-* **Cluster Token** (*required*): base64 decoded JWT token needed by the service account to perform operations on the cluster.  
-  To extract it, refer to [this paragraph](/development_suite/clusters-management/clusters-overview-setup.md#ca-and-token);
-* **Certificate authority**: base64 decoded CA certificate for the TLS connection with the cluster.  
+* **Credentials Type** (*required*): the type of credentials used to authenticate requests to the cluster. It can be one of "Token," "Client credentials," and "Client credentials certificate".
+
+Depending on the credentials type, the other information to be entered will be:
+
+* For credentials of the type "Token":
+  - **Service account token** (*required*): JWT token needed by the service account to perform operations on the cluster.
   To extract it, refer to [this paragraph](/development_suite/clusters-management/clusters-overview-setup.md#ca-and-token).
+* For credentials of the type "Client credentials":
+  - **Access token URL** (*required*): the reference OAuth2 authorization endpoint URL that, to requests that include client ID and client secret, responds by generating temporary credentials;
+  - **Client ID** (*required*): OAuth2 public client identifier;
+  - **Client secret** (*required*): OAuth2 secret, known only to the client and the authorization server;
+  - **Scope** (*required*): permission set used to grant access to specific resource functionalities. In other words, a scope represents what an application can access on behalf of the user.
+* For credentials of the type "Client credentials certificate":
+  - **Access token URL** (*required*): the reference OAuth2 authorization endpoint URL that, to requests that include client ID and client secret, responds by generating temporary credentials;
+  - **Client ID** (*required*): OAuth2 public client identifier;
+  - **Private key** (*required*): private key of the client certificate needed during the SSL handshake to prove that the client owns the certificate;
```

Comment:
It is not during the SSL handshake, it is used to sign an assertion jwt which is used as authentication token